### ISSUE TITLE
Make Precision + Unit attributes public in Python Wrapper

### ISSUE THREAD

**Issue by gambino19** [heitzmann/gdstk#28]
In Documentation for gdstk.Library, precision + unit are listed as attributes for Library object. In using python wrapper, it is not possible to access those attributes
```python
lib = gdstk.Library(precision=1e-10)
dir(lib)
>>> [... dundermethods...,
 'add',
 'cells',
 'delete_property',
 'get_property',
 'name',
 'new_cell',
 'properties',
 'remove',
 'set_property',
 'top_level',
 'write_gds',
 'write_oas'] #missing precision + unit
```
For the cases```gdstk.read_oas``` and ```gdstk.read_gds```, it would be nice to access those attributes from constructed library to compare against results from methods like ```gdstk.oas_precision```

**Comment by heitzmann** [comment]
This is already available in b8b36442f281ca4da6bd4b85a4ca1b20d89ca092, it's just not yet released.

**Comment by gambino19** [comment]
I see, thanks. Will close and update when released.
